index.d.ts

```diff
@@ -206,3 +211,5 @@ export default class Sound {
    */
   isPlaying(): Promise<boolean>
 }
+
+export = Sound;
```

Comment:
I think we also need to add the information about `setSpeed` (Android version condition), `isPlaying`, `isHeadsetPlugged`, `setStreamMute`, `registerHeadsetPlugChangeListener`, `unregisterHeadsetPlugChangeListener`, `setCategory` (added bluetooth option)

Reply:
Made issue: #28.